--- gdb-7.6.2/gdb/sol-thread.c.orig	2013-12-26 22:02:08.930437615 +0000
+++ gdb-7.6.2/gdb/sol-thread.c	2013-12-26 22:05:06.506977611 +0000
@@ -588,6 +588,10 @@
   td_err_e err;
   ptid_t ptid;
 
+  /* Don't attempt to use thread_db for remote targets.  */
+  if (!(target_can_run (&current_target) || core_bfd))
+    return;
+
   /* Do nothing if we couldn't load libthread_db.so.1.  */
   if (p_td_ta_new == NULL)
     return;
